frontend/src/components/WithdrawForm.vue

```diff
@@ -0,0 +1,107 @@
+<template>
```

Comment:
Ah a new component! Totally fine pulling things out into this (that component was getting too big anyway), but it seems something isn't being passed right here cause the receive/withdrawal issues mentioned above